doc/src/site/sphinx/ml/sw_grid.rst

```diff
@@ -45,6 +45,13 @@ H2ODRF in both languages.
             import ai.h2o.sparkling.ml.algos.H2ODRF
             val algo = new H2ODRF().setLabelCol("CAPSULE")
 
+        By default, ``H2ODRF`` algorithm distinguishes between a classification and regression problem based on the type of
+        the label column of the training dataset. If the label column is a string column, a classification model will be trained.
+        If the label column is a numeric column, a regression model will be trained. If you want to be more explicit about
```

Comment:
Maybe we can mention that in the case of using Classifier, you can do classification on numeric columns as we -> stress that we have more freedom

Reply:
ok, will do, thanks!